### PR TITLE
Send notification when build is started

### PR DESCRIPTION
**Pull request by naseemkullah**
Hi @Philmod,

This PR makes the function send a notification when the build is started.
Also abstracts some constants to improve readability.
Hope you like!
Huge thanks to @iliasbhal for the help!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ module.exports.subscribe = (event, callback) => {
   const build = module.exports.eventToBuild(event.data.data);
 
   // Skip if the current status is not in the status list.
-  const status = module.exports.status || ['SUCCESS', 'FAILURE', 'INTERNAL_ERROR', 'TIMEOUT'];
+  const status = module.exports.status || ['WORKING', 'SUCCESS', 'FAILURE', 'INTERNAL_ERROR', 'TIMEOUT'];
```
Comment (Philmod) [inline index.js]: I'd rather leave the user the choice by setting `export GC_SLACK_STATUS="SUCCESS FAILURE TIMEOUT INTERNAL_ERROR"` (see readme)

File: index.js [hunk 2]
```diff
@@ -24,65 +24,81 @@ module.exports.subscribe = (event, callback) => {
...
 };
 
 // createSlackMessage create a message from a build object.
 module.exports.createSlackMessage = (build) => {
-  let message = {
-    text: `Build \`${build.id}\` finished`,
+  const buildFinishTime = new Date(build.finishTime);
+  const buildStartTime = new Date(build.startTime);
+
+  const timestamp = Math.round(buildFinishTime.getTime() / 1000);
```
Comment (Philmod) [inline index.js]: you probably want a different timestamp if `!isWorking`?

**Review by Philmod — CHANGES_REQUESTED** [review]
Thanks for your PR. Please add the appropriate unit tests.

File: index.js (hunk 1 above)
Comment (naseemkullah) [reply index.js]: Sure, good idea!

File: index.js (hunk 2 above)
Comment (naseemkullah) [reply index.js]: Oh true! This should do it!

**Comment by naseemkullah** [comment]
Tests added!
Please let me know if those will suffice.
I've also taken the liberty of updating packages due to vulnerability warnings.

File: package.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "google-container-slack",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "Slack integration for Google Cloud Container Builder, using Google Cloud Functions",
   "main": "index.js",
   "scripts": {
-    "test": "node_modules/.bin/mocha --reporter spec",
+    "test": "nyc mocha",
```
Comment (Philmod) [inline package.json]: That would only work if you installed `nyc` globally. Please roll back.

**Review by Philmod — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (naseemkullah) [reply package.json]: Ah true, thanks.
I've rolled back to using the `node_modules/.bin/` also included the local `nyc`, is that ok?

**Comment by Philmod** [comment]
Thanks.

**Comment by naseemkullah** [comment]
My pleasure, thank you.

**Pull request merged**